src/Bridge/Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -175,26 +175,18 @@ protected function addWhereByStrategy(string $strategy, QueryBuilder $queryBuild
         $valueParameter = ':'.$queryNameGenerator->generateParameterName($field);
         $aliasedField = sprintf('%s.%s', $alias, $field);
 
-        if (self::STRATEGY_EXACT === $strategy) {
-            if (1 === \count($values)) {
+        if (null == $strategy || self::STRATEGY_EXACT == $strategy) {
+            if (1 == \count($values)) {
```

Comment:
We shouldn't revert this part.

Reply:
I don't think so, 2 patches are on this file: 
https://github.com/api-platform/core/commit/9df5a5e3fe438cd265c20bd330f82bce8ef6e709 and https://github.com/api-platform/core/commit/414351b14a7a491d1c7ddfaaaa09254f49652b72 none touch this line ? :/